### DISCUSSION TITLE
希望增加非docker方式的部署文档

### DISCUSSION THREAD

**Discussion by cnguu** [Postcatlab/postcat#209] [Ideas]
希望增加非docker的部署方式，例如源码部署

**Comment by cnguu** [comment]
另外
1. 配置了 .env 中的 mysql 地址为 127.0.0.1，但仍然启动了 docker 的 mysql，我并不希望启动它
2. 启动 docker 的 mysql 没有映射到配置对应的 `MYSQL_PORT`，我觉得是一个 bug
3. 登录请求没有使用配置中的 `EOAPI_API_PREFIX`，我觉得也是个 bug
4. 没有二级目录的相关配置，希望访问的路径是 `http://xxx.com/base/xxx`，其中 `base` 是二级目录

**Reply by scarqin** [reply]
@cnguu 意见下次提到 issue，感谢。

上面的 Bug @buqiyuan 确认下。
